web/src/context/Population.svelte

```diff
@@ -1,23 +1,37 @@
 <script lang="ts">
+  import { onMount } from "svelte";
   import {
     FillLayer,
+    GeoJSON,
     hoverStateFilter,
     LineLayer,
     VectorTileSource,
   } from "svelte-maplibre";
   import { SequentialLegend } from "svelte-utils";
-  import { makeRamp, Popup } from "svelte-utils/map";
+  import { emptyGeojson, makeRamp, Popup } from "svelte-utils/map";
   import { HelpButton, layerId } from "../common";
-  import { assetUrl } from "../stores";
+  import { simdColorScale, simdLimits } from "../common/colors";
+  import { assetUrl, backend } from "../stores";
 
   let showSIMD = false;
+  let aggregateSIMDToNeighbourhoods = false;
   let showDensity = false;
 
-  // Color ramp from https://www.ons.gov.uk/census/maps/choropleth, dark to light.
-  let simdColorScale = ["#080C54", "#186290", "#1F9EB7", "#80C6A3", "#CDE594"];
-  // The percentiles are [1, 100]. The 5 colors cover 4 each.
-  let simdLimits = [0, 20, 40, 60, 80, 100];
-
+  $: neighbourhoods = emptyGeojson();
+  onMount(() => {
```

Comment:
I'm not sure why this needs to be so complicated, but given that I might delete it I didn't dig too deep yet.